### ISSUE TITLE
[Feature] Add extra difficulty mode?

### ISSUE THREAD

**Issue by pelador** [GrognardsFromHell/TemplePlus#252]
Perhaps this is too ambitious to be possible but could there at some point in the future be an added difficulty level to the game? After all, Ironman mode simply adjusts how the save function works. It would be much better to also have a mode where the enemies have more hitpoints and deal more damage more often.

**Comment by DudeMcDude** [comment]
Yeah I do have some plans for that... gotta finish the Prestige Classes first ;)
